content/spin/cli-reference.md

```diff
@@ -887,10 +1014,85 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
+{{ startTab "v1.4"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin help
+
+The Spin CLI
+
+USAGE:
+    spin <SUBCOMMAND>
+
+OPTIONS:
+    -h, --help       Print help information
+    -V, --version    Print version information
+
+SUBCOMMANDS:
+    add          Scaffold a new component into an existing application
+    build        Build the Spin application
+    cloud*       Commands for publishing applications to the Fermyon Cloud.
+    deploy       Package and upload an application to the Fermyon Cloud.
+    doctor       Detect and fix problems with Spin applications
+    help         Print this message or the help of the given subcommand(s)
+    js2wasm*     A plugin to convert js files to Spin compatible modules
+    login        Log into the Fermyon Cloud.
+    new          Scaffold a new application based on a template
+    plugins      Install/uninstall Spin plugins
+    py2wasm*     A plugin to convert Python applications to Spin compatible modules
+    registry     Commands for working with OCI registries to distribute applications
+    templates    Commands for working with WebAssembly component templates
+    up           Start the Spin application
+    watch        Build and run the Spin application, rebuilding and restarting it when files
+                     change
+
+* implemented via plugin
+```
+
+{{ blockEnd }}
+
 {{ blockEnd }}
 
 > Please note: Spin `help` is a convenient way to access help using a subcommand, instead of using the `--help` option. For example, `spin help cloud` will give you the same output as `spin cloud --help`. Similarly, `spin help build` will give you the same output as `spin build --help` and so forth.
 
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin js2wasm
+
+{{ tabs "spin-version" }}
+
+{{ startTab "v1.4"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin js2wasm --help
+
+js2wasm 0.4.0
+A spin plugin to convert javascript files to Spin compatible modules
+
+USAGE:
+    js2wasm [OPTIONS] <input>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -o <output>         [default: index.wasm]
+
+ARGS:
+    <input> 
+
+```
+
+{{ blockEnd }}
+
+{{ blockEnd }}
+
+> As denoted by the `*` when using the `spin --help` command, [js2wasm](/spin/javascript-components) is implemented via plugin. This is why `spin js2wasm --help` above shows slightly different commands (relative to the other commands that see on this page).
```

Comment:
Sorry, I wasn't clear - it's why **Spin** help shows different commands depending on what the user has installed.